Form layout—Documentation (HDS-4886) (#2947)
hashicorp/design-system: website/docs/components/form/layout/partials/guidelines/overview.md

> The Form layout components are used as structural components to organize and space the structure and spacing between elements in forms. 

Author: I would delete the sentence currently here and more the below content to this file:

```
## Usage

Form layout components in Ember and their related Layout “Template” component in Figma are structural components that provide the scaffolding for building consistent forms across products. These components establish spacing and layout standards that give users an experience focused on simplicity and ease of use.

### When to use

- To structure and organize form elements and inputs into a consistent layout
- When creating a form of any size in our products, including forms nested within Flyout or Modal components

### When not to use

- To organize non-form content
- To create key-value pairs, use the [Key Value Inputs](/components/forms/key-value-inputs) component
```

> ## Usage
> 

Reviewer: ```suggestion
```

> Form layout components in Ember and their related Layout “Template” component in Figma are structural components that provide the scaffolding for building consistent forms across products. These components establish spacing and layout standards that give users an experience focused on simplicity and ease of use.

Author: ```suggestion
Form layout components in Ember and the related Form “Template” component in Figma are structural components that provide the scaffolding for building standardized forms across products. These components establish spacing and layout standards to give users a consistent experience.
```
